eZ/Publish/Core/Limitation/ParentContentTypeLimitationType.php

```diff
@@ -234,4 +235,26 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ValueObject $contentInfo
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Location[]
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    private function loadParentLocations(ValueObject $contentInfo)
+    {
+        $isRootChecker = new IsRoot();
+
+        $locations = $this->persistence->locationHandler()->loadLocationsByContent($contentInfo->id);
+        $parentLocations = [];
+        foreach ($locations as $location) {
+            if (!$isRootChecker->isSatisfiedBy($location)) {
+                $parentLocations[] = $this->persistence->locationHandler()->load($location->parentId);
```

Comment:
Either this is a case of need to create a "loadParentLocations()" method
Or at least for v2.4 and up we should rather use `loadList()` to load all parents at once.

Reply:
> Either this is a case of need to create a "loadParentLocations()" method

I like that idea, definitely more efficient. Let's keep in mind this is a CR bugfix, so leaving this decision up to @pawbuj

Reply:
I can handle on merge to just slightly adjust to use loadList 👍